backend/packages/Upgrade/src/api/middlewares/ClientLibMiddleware.ts

```diff
@@ -54,6 +53,7 @@ export class ClientLibMiddleware implements ExpressMiddlewareInterface {
           throw error;
         }
       } else {
+        this.log.info('Next');
```

Comment:
Remove this line.  Not giving any information here

Reply:
done